src/OpenLoco/Windows/ToolTip.cpp

```diff
@@ -66,11 +64,12 @@ namespace OpenLoco::Ui::Windows::ToolTip
     {
         StringManager::formatString(_text, stringId, &args);
 
-        gCurrentFontSpriteBase = Font::medium_bold;
+        Gfx::setCurrentFontSpriteBase(Font::medium_bold);
         int16_t strWidth = Gfx::getStringWidthNewLined(_text);
         strWidth = std::max<int16_t>(strWidth, 196);
 
-        gCurrentFontSpriteBase = Font::medium_bold;
+        // same method called 3 lines up?
```

Comment:
Yes this is needed as the global might have been set to something else by the `getStringWidthNewLined` function.